Fix build with ffmpeg/libav trunk after the big bump.

Index: k9copy-2.3.7-Source/src/import/k9avidecode.cpp
===================================================================
--- k9copy-2.3.7-Source.orig/src/import/k9avidecode.cpp
+++ k9copy-2.3.7-Source/src/import/k9avidecode.cpp
@@ -145,7 +145,7 @@ bool k9AviDecode::open(const QString & _
 // Find the first video stream
     m_videoStream=-1;
     for (i=0; i<m_FormatCtx->nb_streams; i++)
-        if (m_FormatCtx->streams[i]->codec->codec_type==CODEC_TYPE_VIDEO) {
+        if (m_FormatCtx->streams[i]->codec->codec_type==AVMEDIA_TYPE_VIDEO) {
             m_videoStream=i;
             break;
         }
